### PR TITLE
Add --force flag to mysqldump

### PR DESCRIPTION
**Pull request by k4p7**
This will let us ignore constructs we have not been GRANTed permission
to see and continue on with the dump and return with exit code 2

### CODE REVIEW

File: devdbsetup [hunk 1]
```diff
@@ -105,7 +106,7 @@ do
 done
 PASS="$(security find-generic-password -a "$(whoami)" -s "$PASS_KEY" -w)"
 
-createRemoteSchemaDump
+createRemoteSchemaDump || true
```
Comment (karel-kahula) [inline devdbsetup:109]: hide that `|| true` bad boy inside the function instead.

Comment (karel-kahula) [inline devdbsetup]: hide that || true bad boy inside the function instead.

**Review by karel-kahula — CHANGES_REQUESTED** [review]

File: devdbsetup [hunk 2]
```diff
@@ -26,6 +26,7 @@ createRemoteSchemaDump() {
     --skip-add-locks \
     --skip-disable-keys \
     --column-statistics=0 \
+    --force \
```
Comment (karel-kahula) [inline devdbsetup]: i would prefer if we added a flag to devdbsetup so we could toggle this flag

**Review by karel-kahula — CHANGES_REQUESTED** [review]

File: devdbsetup (hunk 1 above)
Comment (k4p7) [reply devdbsetup]: done

File: devdbsetup (hunk 2 above)
Comment (k4p7) [reply devdbsetup]: done

**Review by karel-kahula — APPROVED** [review]
nice job man. looks good!

**Pull request merged**